api/client/trust.go

```diff
@@ -199,14 +199,16 @@ func (cli *DockerCli) getPassphraseRetriever() passphrase.Retriever {
 	// Backwards compatibility with old env names. We should remove this in 1.10
 	if env["root"] == "" {
 		env["root"] = os.Getenv("DOCKER_CONTENT_TRUST_OFFLINE_PASSPHRASE")
-		fmt.Fprintf(cli.err, "[DEPRECATED] The environment variable DOCKER_CONTENT_TRUST_OFFLINE_PASSPHRASE has been deprecated and will be removed in v1.10. Please use DOCKER_CONTENT_TRUST_ROOT_PASSPHRASE\n")
-
+		if os.Getenv("DOCKER_CONTENT_TRUST_OFFLINE_PASSPHRASE") != "" {
```

Comment:
Updated. Changed for these if clauses, didn't change rest of the file because that is not what this PR fixes.